source/administration/upgrade.md

```diff
@@ -11,19 +11,17 @@ To start, select one of the following guides:
 
 ## Upgrade Team Edition to 3.1.x and later
 
-**Important Note:** Security-related changes were made in 3.8 that require you to verify settings in the System Console before upgrading from version 3.7.x and earlier to any version greater than 3.8.0.
-
-**To prepare your system when upgrading from 3.7.x and earlier**:
-
-  1. In the System Console, go to **General > Configuration** and make sure that the **Site URL** is specified. It must not be empty.
-  2. In the System Console, go to **General > Logging** and make sure that the **File Log Directory** field is either empty or has a directory path only. It must not have a filename as part of the path.
+**Important Note:** Security-related changes require you to specify the Site URL setting when upgrading to `v3.6.x` and later. The Site URL can be set in the `config.json` or in **System Console** > **General > Configuration** > [**Site URL**](https://docs.mattermost.com/administration/config-settings.html#site-url). If you are using GitLab Mattermost, you must set the Site URL in the `gitlab.rb` file.
 
 **To upgrade your system**:
 
 1. Download the **appropriate next upgrade** of your Team Edition server and note any compatibility procedures:
       1. Run `platform -version` to check the current version of your Mattermost server
       2. Determine the appropriate next upgrade for your server:
-          - Mattermost `v3.0.x` and later can upgrade directly to the [latest release of Mattermost](https://about.mattermost.com/download/), but see the previous note about upgrading from 3.7.x and earlier.
+          - Mattermost `v3.0.x` and later can upgrade directly to the [latest release of Mattermost](https://about.mattermost.com/download/) considering the conditions below:
+              - Note: Upgrading to `v3.9.x` and later will invalidate existing team invite links, password reset links, and email verification links.
+              - Note: Upgrading to `v3.8.x` and later, if you're using NGINX as a proxy for the Mattermost Server, replace the `location /api/v3/users/websocket {` line with `location ~ /api/v[0-9]+/(users/)?websocket$ {` in the `/etc/nginx/sites-available/mattermost` NGINX configuration file. [See documentation to learn more](https://docs.mattermost.com/install/install-ubuntu-1404.html#configuring-nginx-as-a-proxy-for-mattermost-server).
```

Comment:
It's 3.8 - I submitted a separate PR so that it's already included in the existing upgrade guide: https://github.com/mattermost/docs/pull/1144